tests/Microsoft.Identity.Test.SideBySide/Microsoft.Identity.Test.SideBySide.csproj

```diff
@@ -6,10 +6,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="4.5.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="4.5.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
     <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
```

Comment:
> <PackageReference Include="Newtonsoft.Json" Version="12.0.1" /> [](start = 3, length = 64)

is this required?

Reply:
It is for now since adal requires newtonsoft json.  if it's not there, the tests fail that it can't load the dll.  once we have updated adal with embedded json i think we can remove this.